docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -598,50 +700,487 @@ Confirm Network Functionality
+      be run on the Sawtooth node in pod 0. (The actual prompt is similar to
+      ``root@sawtooth-0-5ff6d9d578-5w45k:/#``.)
+
+#. Display the list of blocks on the Sawtooth blockchain.
+
+     .. code-block:: none
+
+        root@sawtooth-0# sawtooth block list
+
+   The output will be similar to this example:
+
+     .. code-block:: console
+
+        NUM  BLOCK_ID                                                                                                                          BATS  TXNS  SIGNER
+        2    f40b90d06b4a9074af2ab09e0187223da7466be75ec0f472f2edd5f22960d76e402e6c07c90b7816374891d698310dd25d9b88dce7dbcba8219d9f7c9cae1861  3     3     02e56e...
+        1    4d7b3a2e6411e5462d94208a5bb83b6c7652fa6f4c2ada1aa98cabb0be34af9d28cf3da0f8ccf414aac2230179becade7cdabbd0976c4846990f29e1f96000d6  1     1     034aad...
+        0    0fb3ebf6fdc5eef8af600eccc8d1aeb3d2488992e17c124b03083f3202e3e6b9182e78fef696f5a368844da2a81845df7c3ba4ad940cee5ca328e38a0f0e7aa0  3     11    034aad...
+
+   Block 0 is the :term:`genesis block`. The other two blocks contain
+   transactions for on-chain settings, such as setting PoET consensus.
+
+#. In a separate terminal window, connect to a different pod (such as pod 1) and
+   verify that it has joined the Sawtooth network.
+
+     .. code-block:: none
+
+        $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-1/{print $1}') --container sawtooth-shell -- bash
+
+        root@sawtooth-1#
+
+   .. note::
+
+      The prompt ``root@sawtooth-1#`` marks the commands that should be run on
+      the Sawtooth node in pod 1.
+
+#. Display the list of blocks on the second pod.
+
+     .. code-block:: none
+
+        root@sawtooth-1# sawtooth block list
+
+   You should see the same list of blocks with the same block IDs, as in this
+   example:
+
+     .. code-block:: console
+
+        NUM  BLOCK_ID                                                                                                                          BATS  TXNS  SIGNER
+        2    f40b90d06b4a9074af2ab09e0187223da7466be75ec0f472f2edd5f22960d76e402e6c07c90b7816374891d698310dd25d9b88dce7dbcba8219d9f7c9cae1861  3     3     02e56e...
+        1    4d7b3a2e6411e5462d94208a5bb83b6c7652fa6f4c2ada1aa98cabb0be34af9d28cf3da0f8ccf414aac2230179becade7cdabbd0976c4846990f29e1f96000d6  1     1     034aad...
+        0    0fb3ebf6fdc5eef8af600eccc8d1aeb3d2488992e17c124b03083f3202e3e6b9182e78fef696f5a368844da2a81845df7c3ba4ad940cee5ca328e38a0f0e7aa0  3     11    034aad...
+
+#. (Optional) You can repeat the previous two steps on the other pods to verify
+   that they have the same block list. To connect to a different pod, replace
+   the `N` (in ``sawtooth-N``) in the following command with the pod number.
+   command:
+
+     .. code-block:: none
+
+        $ kubectl exec -it $(kubectl get pods | awk '/sawtooth-N/{print $1}') --container sawtooth-shell -- bash
+
+#. (Optional) You can also connect to the shell container of any pod, and
```

Comment:
This is rendering bold and indenting oddly. It might be caused by an extra space on the next line.

Reply:
You're right - it was an extra space on the next line.  Fixed.